package.json

```diff
@@ -11,7 +11,7 @@
     "rollup": "^4.34.1",
     "source-map": "0.7.4",
     "typescript": "^5.7.3",
-    "vite": "^6.0.11",
+    "vite": "^6.1.0",
```

Comment:
It's important we bump Vite to at least `6.1.0` for the test build.

We introduced a fix to Vite in https://github.com/vitejs/vite/pull/19010 to allow dynamic fields in the literal worker options. This allows Emscripten to keep on using `name: 'em-pthread-' + PThread.nextWorkerID,` for debugging.

Reply:
Nice!  I just updated our deps 3 days ago in #23574, but I'll do it again now as a separate PR I think,

Reply:
https://github.com/emscripten-core/emscripten/pull/23610

Reply:
Awesome, thanks! I've synced those changes in.